docs/src/pages/demos/badges/badges.md

```diff
@@ -33,12 +33,9 @@ The badge auto hides with badgeContent is zero. You can override this with the `
 
 {{"demo": "pages/demos/badges/BadgeVisibility.js"}}
 
-## Customized Badge
+## Customized badges
 
-If you have been reading the [overrides documentation page](/customization/overrides/)
-but you are not confident jumping in,
-here is one example of how you can change the badge position.
+If you have read the [overrides documentation page](/customization/overrides/)
+but you are not confident jumping in, here are some examples of how you can customize the component.
```

Comment:
```suggestion
and you are still not confident jumping in, here are some examples of how you can customize the component.
```
?

Reply:
👌

Reply:
Perhaps a hybrid?

```suggestion
but you are still not confident jumping in, here are some examples of how you can customize the component.
```

OTOH, I've never been to keen on the "not confident" – it's a bit condescending and "jumping in" feels like a misplaced idiom.

How about something like:

"Here are some examples of customizing the component. You can learn more about this in the [overrides documentation page](/customization/overrides/)."

A forward reference seems more natural than an assumptive back ref.

The "So you've been wondering X?" type phrases dotted around the docs feel a bit stilted in English. (No offence @oliviertassinari. Perhaps that's idiomatic in French?)

Reply:
Hmm, I did have the same worry. I like your change :+1: